platform/core/flatbuffers_shared/generate_flatbuffers.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -e    # Exit if any command fails
+
+echo '===> Compiling the Schema...'
+
+cd flatbuffers_shared
+# TODO: Add iOS and Android folders
+RUST_FLATBUFFERS_OUT=../src/flatbuffers_generated
+FLATBUFFER_MODELS=$( find -P flatbuffer_models -name "*.fbs" )
+
+echo "*** Removing previously compiled models ***"
+rm -rf $RUST_FLATBUFFERS_OUT && mkdir $RUST_FLATBUFFERS_OUT
```

Comment:
I've been bitten so many times with this kind of things that I live in fear every time I do `rm -rf`. Trying to glob for just the files that are generated might be a bit safer...